Revamp Iteration realted BBEs (#3910)
ballerina-platform/ballerina-distribution: examples/break-statement/break_statement.md

> # Break Statement

Reviewer: ```suggestion
# Break statement
```

# Break Statement

> `Break` statements can be used to terminate the execution within a nearest enclosing

Reviewer 2: ```suggestion
A `break` statement can be used to terminate the execution within the nearest enclosing `while` or `foreach` statement.
```

> `Break` statements can be used to terminate the execution within a nearest enclosing
> `while` and `foreach` statement.

Reviewer 2: ```suggestion
A `break` statement can be used to terminate the execution within the nearest enclosing `while` or `foreach` statement.
```

Reviewer 2: You can write the entire description in a single line unless you need them in separate paragraphs.  
If you want to start a new paragraph use a new line. If we add new lines for each and every sentence they'll be rendered as if they are different paragraphs on the website.

# Break Statement

`Break` statements can be used to terminate the execution within a nearest enclosing
`while` and `foreach` statement.

::: code break_statement.bal :::

> ::: out break_statement.out :::

Reviewer 3: Add related Links, continue, while, foreach.
